### PR TITLE
Samsung Internet, Updating O-T files, in api/

### PR DESCRIPTION
**Pull request by AdaRoseCannon**
A checklist to help your pull request get merged faster:
- [ ] Summarize your changes
- [ ] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)
- [ ] Data: if you tested something, describe how you tested with details like browser and version
- [ ] Review the results of the linter and fix problems reported (If you need help, please ask in a comment!)
- [ ] Link to related issues or pull requests, if any

### CODE REVIEW

File: api/ScreenOrientation.json [hunk 1]
```diff
@@ -82,7 +82,7 @@
               "version_added": false
             },
             "samsunginternet_android": {
-              "version_added": true
+              "version_added": "5.0"
```
Comment (vinyldarkscratch) [inline api/ScreenOrientation.json]:
```suggestion
              "version_added": "3.0"
```

File: api/ScreenOrientation.json [hunk 2]
```diff
@@ -226,7 +226,7 @@
...
```
Comment (vinyldarkscratch) [inline api/ScreenOrientation.json]:
```suggestion
              "version_added": "3.0"
```

File: api/TextDecoder.json [hunk 3]
```diff
@@ -124,7 +124,7 @@
               "version_added": "10.3"
             },
             "samsunginternet_android": {
-              "version_added": null
+              "version_added": "4.0"
```
Comment (vinyldarkscratch) [inline api/TextDecoder.json]:
```suggestion
              "version_added": "3.0"
```

File: api/TextDecoder.json [hunk 4]
```diff
@@ -59,7 +59,7 @@
             "version_added": "10.3"
           },
           "samsunginternet_android": {
-            "version_added": true
+            "version_added": "4.0"
```
Comment (vinyldarkscratch) [inline api/TextDecoder.json]:
```suggestion
            "version_added": "3.0"
```

File: api/TextDecoder.json [hunk 5]
```diff
@@ -195,7 +195,7 @@
               "version_added": "10.3"
             },
             "samsunginternet_android": {
-              "version_added": true
+              "version_added": "4.0"
```
Comment (vinyldarkscratch) [inline api/TextDecoder.json]:
```suggestion
              "version_added": "3.0"
```

File: api/TextDecoder.json [hunk 6]
```diff
@@ -260,7 +260,7 @@
...
```
Comment (vinyldarkscratch) [inline api/TextDecoder.json]:
```suggestion
              "version_added": "3.0"
```

File: api/TextDecoder.json [hunk 7]
```diff
@@ -413,7 +413,7 @@
               "version_added": "10.3"
             },
             "samsunginternet_android": {
-              "version_added": null
+              "version_added": "4.0"
```
Comment (vinyldarkscratch) [inline api/TextDecoder.json]:
```suggestion
              "version_added": "3.0"
```

File: api/TextEncoder.json [hunk 8]
```diff
@@ -156,7 +156,7 @@
               "version_added": "10.3"
             },
             "samsunginternet_android": {
-              "version_added": null
+              "version_added": "4.0"
```
Comment (vinyldarkscratch) [inline api/TextEncoder.json]:
```suggestion
              "version_added": "3.0"
```

File: api/TextEncoder.json [hunk 9]
```diff
@@ -219,7 +219,7 @@
               "version_added": "10.3"
             },
             "samsunginternet_android": {
-              "version_added": true
+              "version_added": "4.0"
```
Comment (vinyldarkscratch) [inline api/TextEncoder.json]:
```suggestion
              "version_added": "3.0"
```

File: api/TextEncoder.json [hunk 10]
```diff
@@ -333,7 +333,7 @@
...
```
Comment (vinyldarkscratch) [inline api/TextEncoder.json]:
```suggestion
              "version_added": "3.0"
```

File: api/TextEncoder.json [hunk 11]
```diff
@@ -384,7 +384,7 @@
               "version_added": "10.3"
             },
             "samsunginternet_android": {
-              "version_added": null
+              "version_added": "4.0"
```
Comment (vinyldarkscratch) [inline api/TextEncoder.json]:
```suggestion
              "version_added": "3.0"
```

**Comment by AdaRoseCannon** [comment]
Fixes applied, version 11 just landed, do you think it would be best to use your tool to try out the update?

**Comment by AdaRoseCannon** [comment]
```
✖ api/ScreenOrientation.json

  Consistency - 1 error:

  → 1 × ScreenOrientation [api.ScreenOrientation]: 

    → Basic support in samsunginternet_android was declared implemented in a later version (5.0) than the following sub-feature(s):

      → api.ScreenOrientation.angle: 3.0

      → api.ScreenOrientation.lock: 3.0

      → api.ScreenOrientation.onchange: 3.0

      → api.ScreenOrientation.type: 3.0

      → api.ScreenOrientation.unlock: 3.0
```
unlock:
```
            "samsunginternet_android": {
              "version_added": "5.0"
            },
```

**Comment by AdaRoseCannon** [comment]
Ah fixed :)

**Comment by vinyldarkscratch** [comment]
> Fixes applied, version 11 just landed, do you think it would be best to use your tool to try out the update?

Ah, thanks for mentioning!  I just created a pull request to get the browser version added.  :wink:  I'm thinking we should get the mirroring taken care of in another PR after that one's been merged, but we can use whichever script you prefer!

File: api/PerformanceFrameTiming.json [hunk 12]
```diff
@@ -37,6 +37,9 @@
           "safari_ios": {
             "version_added": false
           },
+          "samsunginternet_android": {
+            "version_added": false
```
Comment (vinyldarkscratch) [inline api/PerformanceFrameTiming.json]:
```suggestion
            "version_added": false,
            "notes": "See <a href='https://crbug.com/120796'>Chrome bug 120796</a>"
```

File: api/Performance.json [hunk 13]
```diff
@@ -103,6 +106,16 @@
             "safari_ios": {
               "version_added": "11"
             },
+            "samsunginternet_android": [
+              {
+                "version_added": "3.0"
+              },
+              {
+                "prefix": "webkit",
+                "version_added": "2.0",
```
Comment (vinyldarkscratch) [inline api/Performance.json]:
```suggestion
                "version_added": "1.5",
```

File: api/Performance.json [hunk 14]
```diff
@@ -103,6 +106,16 @@
             "safari_ios": {
               "version_added": "11"
             },
+            "samsunginternet_android": [
+              {
+                "version_added": "3.0"
```
Comment (vinyldarkscratch) [inline api/Performance.json]:
```suggestion
                "version_added": "2.0"
```

File: api/Performance.json [hunk 15]
```diff
@@ -103,6 +106,16 @@
...
+              },
+              {
+                "prefix": "webkit",
+                "version_added": "2.0",
+                "version_removed": "3.0"
```
Comment (vinyldarkscratch) [inline api/Performance.json]:
```suggestion
                "version_removed": "2.0"
```

File: api/Performance.json [hunk 16]
```diff
@@ -165,6 +178,16 @@
             "safari_ios": {
               "version_added": "11"
             },
+            "samsunginternet_android": [
+              {
+                "version_added": "3.0"
+              },
+              {
+                "prefix": "webkit",
+                "version_added": "2.0",
```
Comment (vinyldarkscratch) [inline api/Performance.json]:
```suggestion
                "version_added": "1.5",
```

File: api/Performance.json [hunk 17]
```diff
@@ -165,6 +178,16 @@
             "safari_ios": {
               "version_added": "11"
             },
+            "samsunginternet_android": [
+              {
+                "version_added": "3.0"
```
Comment (vinyldarkscratch) [inline api/Performance.json]:
```suggestion
                "version_added": "2.0"
```

File: api/Performance.json [hunk 18]
```diff
@@ -165,6 +178,16 @@
...
+              },
+              {
+                "prefix": "webkit",
+                "version_added": "2.0",
+                "version_removed": "3.0"
```
Comment (vinyldarkscratch) [inline api/Performance.json]:
```suggestion
                "version_removed": "2.0"
```

File: api/Performance.json [hunk 19]
```diff
@@ -227,6 +250,16 @@
             "safari_ios": {
               "version_added": "11"
             },
+            "samsunginternet_android": [
+              {
+                "version_added": "5.0"
+              },
+              {
+                "prefix": "webkit",
+                "version_added": "2.0",
```
Comment (vinyldarkscratch) [inline api/Performance.json]:
```suggestion
                "version_added": "1.5",
```

File: api/Performance.json [hunk 20]
```diff
@@ -296,6 +329,16 @@
             "safari_ios": {
               "version_added": "11"
             },
+            "samsunginternet_android": [
+              {
+                "version_added": "2.0"
```
Comment (vinyldarkscratch) [inline api/Performance.json]:
```suggestion
                "version_added": "1.5"
```

File: api/Performance.json [hunk 21]
```diff
@@ -296,6 +329,16 @@
...
+              },
+              {
+                "prefix": "webkit",
+                "version_added": "2.0",
```
Comment (vinyldarkscratch) [inline api/Performance.json]:
```suggestion
                "version_added": "1.5",
```

File: api/Performance.json [hunk 22]
```diff
@@ -296,6 +329,16 @@
...
+                "version_removed": "4.0"
```
Comment (vinyldarkscratch) [inline api/Performance.json]:
```suggestion
                "version_removed": "3.0"
```

File: api/Performance.json [hunk 23]
```diff
@@ -358,6 +401,16 @@
             "safari_ios": {
               "version_added": "11"
             },
+            "samsunginternet_android": [
+              {
+                "version_added": "2.0"
```
Comment (vinyldarkscratch) [inline api/Performance.json]:
```suggestion
                "version_added": "1.5"
```

File: api/Performance.json [hunk 24]
```diff
@@ -358,6 +401,16 @@
...
+              },
+              {
+                "prefix": "webkit",
+                "version_added": "2.0",
```
Comment (vinyldarkscratch) [inline api/Performance.json]:
```suggestion
                "version_added": "1.5",
```

File: api/Performance.json [hunk 25]
```diff
@@ -358,6 +401,16 @@
...
+                "version_removed": "4.0"
```
Comment (vinyldarkscratch) [inline api/Performance.json]:
```suggestion
                "version_removed": "3.0"
```

File: api/Performance.json [hunk 26]
```diff
@@ -434,6 +487,16 @@
             "safari_ios": {
               "version_added": "11"
             },
+            "samsunginternet_android": [
+              {
+                "version_added": "2.0"
```
Comment (vinyldarkscratch) [inline api/Performance.json]:
```suggestion
                "version_added": "1.5"
```

File: api/Performance.json [hunk 27]
```diff
@@ -434,6 +487,16 @@
...
+              },
+              {
+                "prefix": "webkit",
+                "version_added": "2.0",
```
Comment (vinyldarkscratch) [inline api/Performance.json]:
```suggestion
                "version_added": "1.5",
```

File: api/Performance.json [hunk 28]
```diff
@@ -434,6 +487,16 @@
...
+                "version_removed": "4.0"
```
Comment (vinyldarkscratch) [inline api/Performance.json]:
```suggestion
                "version_removed": "3.0"
```

File: api/Performance.json [hunk 29]
```diff
@@ -496,6 +559,16 @@
             "safari_ios": {
               "version_added": "11"
             },
+            "samsunginternet_android": [
+              {
+                "version_added": true
```
Comment (vinyldarkscratch) [inline api/Performance.json]:
```suggestion
                "version_added": "1.5
```

File: api/Performance.json [hunk 30]
```diff
@@ -496,6 +559,16 @@
...
+              },
+              {
+                "prefix": "webkit",
```
Comment (vinyldarkscratch) [inline api/Performance.json]: We should remove this support statement; the prefixed version was added in Chrome 25 and removed in Chrome 28, which both equate to Samsung Internet 1.5 (so it was never in a Samsung Internet version).

File: api/Performance.json [hunk 31]
```diff
@@ -558,6 +631,16 @@
...
```
Comment (vinyldarkscratch) [inline api/Performance.json]: We should remove this support statement; the prefixed version was added in Chrome 25 and removed in Chrome 28, which both equate to Samsung Internet 1.5 (so it was never in a Samsung Internet version).

File: api/Performance.json [hunk 32]
```diff
@@ -558,6 +631,16 @@
             "safari_ios": {
               "version_added": "11"
             },
+            "samsunginternet_android": [
+              {
+                "version_added": true
```
Comment (vinyldarkscratch) [inline api/Performance.json]:
```suggestion
                "version_added": "1.5"
```

File: api/Performance.json [hunk 33]
```diff
@@ -719,6 +808,9 @@
             "safari_ios": {
               "version_added": "9"
             },
+            "samsunginternet_android": {
+              "version_added": "2.0"
```
Comment (vinyldarkscratch) [inline api/Performance.json]:
```suggestion
              "version_added": "1.5"
```

File: api/Performance.json [hunk 34]
```diff
@@ -781,6 +873,16 @@
             "safari_ios": {
               "version_added": false
             },
+            "samsunginternet_android": [
+              {
+                "version_added": "5.0"
+              },
+              {
+                "alternative_name": "onwebkitresourcetimingbufferfull",
+                "version_added": "2.0",
```
Comment (vinyldarkscratch) [inline api/Performance.json]:
```suggestion
                "version_added": "1.5",
```

File: api/Performance.json [hunk 35]
```diff
@@ -851,6 +953,16 @@
             "safari_ios": {
               "version_added": false
             },
+            "samsunginternet_android": [
+              {
+                "version_added": "5.0"
+              },
+              {
+                "alternative_name": "webkitresourcetimingbufferfull",
+                "version_added": "2.0",
```
Comment (vinyldarkscratch) [inline api/Performance.json]:
```suggestion
                "version_added": "1.5",
```

File: api/Performance.json [hunk 36]
```diff
@@ -920,6 +1032,16 @@
             "safari_ios": {
               "version_added": false
             },
+            "samsunginternet_android": [
+              {
+                "version_added": "5.0"
+              },
+              {
+                "prefix": "webkit",
+                "version_added": "2.0",
```
Comment (vinyldarkscratch) [inline api/Performance.json]:
```suggestion
                "version_added": "1.5",
```

File: api/Screen.json [hunk 37]
```diff
@@ -426,7 +427,7 @@
               "version_added": false
             },
             "samsunginternet_android": {
-              "version_added": true
+              "version_added": "4.0"
```
Comment (vinyldarkscratch) [inline api/Screen.json]:
```suggestion
              "version_added": "3.0"
```

**Pull request merged**